Boardfiles/nucleol552zeq/Core/Src/usart.c

```diff
@@ -46,7 +44,7 @@ void MX_LPUART1_UART_Init(void)
 
   /* USER CODE END LPUART1_Init 1 */
   hlpuart1.Instance = LPUART1;
-  hlpuart1.Init.BaudRate = 209700;
+  hlpuart1.Init.BaudRate = 230400;

```

Comment:
This is a similar issue as the i2c, the baudrate is changed wiredly

Reply:
I did this so I could print stuff on my terminal on my laptop, will change this back later